app/views/articles/_sidebar.html.erb

```diff
@@ -8,16 +8,42 @@
     <% cache("display-area-left-#{rand(5)}-#{user_signed_in?}", expires_in: 5.minutes) do %>
       <% @left_sidebar_ad = DisplayAd.for_display("sidebar_left", user_signed_in?) %>
       <% if @left_sidebar_ad %>
-        <div class="crayons-card crayons-card--secondary p-3 crayons-sponsorship-widget" data-display-unit data-id="<%= @left_sidebar_ad.id %>">
-          <%= @left_sidebar_ad.processed_html.html_safe %>
+        <div class="crayons-card crayons-card--secondary crayons-sponsorship-widget" data-display-unit data-id="<%= @left_sidebar_ad.id %>">
```

Comment:
It looks like more or less the same code in five places here, I think this needs to be pulled into a partial.

Reply:
@benhalpern I was unable to remove this entire display-ad code to a partial because not all display ads as same in design, especially the below-comments-ad is much different and therefore what I have done is I have taken the entire `header` code of the display ad into a partial. Earlier we had only the contents of drop-down menu into the partial.

Reply:
Update on this @benhalpern I have merged the display ad fully.